Resources/config/jackalope.xml

```diff
@@ -53,7 +53,7 @@
                  public="false"
         >
             <argument type="collection"/>
-            <argument type="service" id="doctrine.dbal.default_connection"/>
+            <argument type="service" id="doctrine_phpcr.jackalope_doctrine_dbal.default_connection"/>
```

Comment:
@danrot yes, thats why i asked you to check on it. what you did was an improvement, but as it turns out not correct in more complicated scenarios (only an issue if the different dbal connections use different parameters, which sounds like an edge case to me).